test/index.js

```diff
@@ -483,6 +483,10 @@ describe('memoizeOne', () => {
     });
   });
 
+  describe('clearing the cache', () => {
```

Comment:
I will write the tests if people think it is a good idea to extend the api like this